src/main/proto/orc/proto/orc_proto.proto

```diff
@@ -84,6 +84,27 @@ message CollectionStatistics {
   optional uint64 total_children = 3;
 }
 
+// Bounding box for Geometry or Geography type in the representation of min/max
+// value pair of coordinates from each axis.
+message BoundingBox {
+  optional double xmin = 1;
+  optional double xmax = 2;
+  optional double ymin = 3;
+  optional double ymax = 4;
+  optional double zmin = 5;
+  optional double zmax = 6;
+  optional double mmin = 7;
+  optional double mmax = 8;
+}
+
+// Statistics specific to Geometry or Geography type
+message GeospatialStatistics {
+  // A bounding box of geospatial instances
+  optional BoundingBox bbox = 1;
```

Comment:
Is `bbox` a well-known name? I'm just wondering if we can use `bounding_box` like the other field names.

Reply:
Yes, `bbox` is a well-known geospatial acronym.